Assets/__Scripts/Platforms/Track Rings/LPRingLaserManager.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SimpleJSON;
+using System.Collections.Generic;
+using UnityEngine;
+
+class LPRingLaserManager : ITrackLaneRingsManager
+{
+
+    [SerializeField] private List<RotatingLightsRandom> lpLasers;
+
+    public override Object[] GetToDestroy()
+    {
+        return new Object[] { this };
+    }
+
+    public override void HandlePositionEvent()
+    {
+        
+    }
+
+    public override void HandleRotationEvent(JSONNode customData = null)
+    {
+        lpLasers.ForEach(it => it.SwitchStyle());
```

Comment:
The left/right rotating lights are not perfectly reflected across the Z axis:

![image](https://user-images.githubusercontent.com/30004286/91127488-3b541500-e65b-11ea-8605-5e11a8835b2a.png)

When in game, the left/right rotating lights are always perfectly reflected across the Z axis: [video](https://www.youtube.com/watch?v=1wdqfmk5hwI)

Reply:
I re-looked at this and I fixed some places where this could happen. It *is* possible to have them not mirrored in game if you set separate laser speeds: 
![image](https://user-images.githubusercontent.com/534069/91167501-f9af8280-e6cb-11ea-8591-38b1950def6c.png) but I'm not sure if them getting re-aligned when ring rotation events happen is a feature or a bug.

```csharp
			_rotationDataL.rotationAngle = _randomStartRotation + _rotationDataL.startRotationAngle;
			_rotationDataL.rotationSpeed = Mathf.Abs(_rotationDataL.rotationSpeed);
			_rotationDataR.rotationAngle = 0f - _randomStartRotation + _rotationDataR.startRotationAngle;
			_rotationDataR.rotationSpeed = 0f - Mathf.Abs(_rotationDataL.rotationSpeed);
```

Seems weird to ignore the previous right laser rotation speed

Reply:
(Now updated to match the game though)